README.md

```diff
@@ -2,207 +2,219 @@
-  onPressArrowLeft={subtractMonth => subtractMonth()}
-  // Handler which gets executed when press arrow icon right. It receive a callback can go next month
-  onPressArrowRight={addMonth => addMonth()}
-  // Disable left arrow. Default = false
-  disableArrowLeft={true}
-  // Disable right arrow. Default = false
-  disableArrowRight={true}
-  // Disable all touch events for disabled days. can be override with disableTouchEvent in markedDates
-  disableAllTouchEventsForDisabledDays={true}
-  // Replace default month and year title with custom one. the function receive a date as parameter
-  renderHeader={date => {
-    /*Return JSX*/
-  }}
-  // Enable the option to swipe between months. Default = false
-  enableSwipeMonths={true}
-/>
+import React, { useState } from 'react';
+import { Calendar, LocaleConfig } from 'react-native-calendars';
+
+const App = () => {
+  const [selected, setSelected] = useState('');
+
+  return (
+    <Calendar
+      style={{
+        borderWidth: 1,
+        borderColor: 'gray',
+        height: 350,
+      }}
+      theme={{
+        backgroundColor: '#ffffff',
+        calendarBackground: '#ffffff',
+        textSectionTitleColor: '#b6c1cd',
+        selectedDayBackgroundColor: '#00adf5',
+        selectedDayTextColor: '#ffffff',
+        todayTextColor: '#00adf5',
+        dayTextColor: '#2d4150',
+        textDisabledColor: '#d9e
 ```
+## Customized Calendar Examples
 
-#### Date marking
+### Calendar
+<kbd>
+  <img src="https://github.com/wix-private/wix-react-native-calendar/blob/master/demo/assets/calendar.gif?raw=true">
+</kbd>
 
-**Disclaimer**: Make sure that `markedDates` param is immutable. If you change `markedDates` object content but the reference to it does not change calendar update will not be triggered.
 
-Dot marking
+### Dot marking
 
 <kbd>
   <img height=50 src="https://github.com/wix-private/wix-react-native-calendar/blob/master/demo/assets/marking1.png?raw=true">
 </kbd>
-<p></p>
-
-```javascript
-<Calendar
-  // Collection of dates that have to be marked. Default = {}
-  markedDates={{
-    '2012-05-16': {selected: true, marked: true, selectedColor: 'blue'},
-    '2012-05-17': {marked: true},
-    '2012-05-18': {marked: true, dotColor: 'red', activeOpacity: 0},
-    '2012-05-19': {disabled: true, disableTouchEvent: true}
-  }}
-/>
-```
-
-You can customize a dot color for each day independently.
 
-Multi-Dot marking
+### Multi-Dot marking
 
 <kbd>
  <img height=50 src="https://github.com/wix-private/wix-react-native-calendar/blob/master/demo/assets/marking4.png?raw=true">
 </kbd>
-<p></p>
-
-Use `markingType={'multi-dot'}` if you want to display more than one dot. Both the `<Calendar/>` and `<CalendarList/>` support multiple dots by using `dots` array in `markedDates` prop.
-The property `color` is mandatory while `key` and `selectedColor` are optional. If key is omitted then the array index is used as key. If `selectedColor` is omitted then `color` will be used for selected dates.
```

Comment:
This is also an important note, no? Or is there another place for it?

Reply:
I moved these from the main repo readme into the docusaurus pages.